/kurome/pino:
  ros__parameters:
    map_out: "map"
    map_publish_interval: 1.0
    publish_visualization: true
    visualization_topic: "pino/visual"
    beacon_in: "beacon"
    odom_in: "odom"
    scan_in: "scan"
    cloud_in: "cloud"
    reset_in: "pino/reset"
    use_odom_vel: true
    vel_in: "cmd_vel"
    time_error: 0.01
    beacon_lost_time: 0.25
    fix_beacon_nodes: true
    estimate_movement_updates: false
    aggregate_sensor_data: false
    aggregation_interval: 0.1
    algorithm: "builtin"
    bin_size: 1.0
    linear_update_dist: 0.3
    angular_update_dist: 0.3
    lidar_acceptance_threshold: 1.1
    point_cloud_acceptance_threshold: 0.7
    node_association_dist: 0.25
    recent_length: 3
    loop_closure_dist: 1

/kurome/yoriko:
  ros__parameters:
    path_topic: "path"
    goal_topic: "/goal_pose"
    position_topic: "odom"
    grid_topic: "map"
    algorithm: "a_star"
    collision_radius: 0.5
    publish_rate: 0.1
    launch_gui: true
    obstacle_threshold: 30
    allow_out_of_bounds_goal: true
    queue_limit: 40000
    backtrack_count: -1

/kurome/misao:
  ros__parameters:
    path_in: "path"
    odom_in: "odom"
    goal_in: "/goal_pose"
    vel_out: "cmd_vel"
    map_in: "map"
    publish_rate: 0.1
    obstacle_threshold: 30
    launch_gui: true
    run_visualization: true
    visualization_topic: "misao/visual"
    simulate_trajectory: false
    trajectory_topic: "misao/trajectory"
    collision_radius: 0.4
    algorithm: "elastic_band"
    band_length: -1
    influence_range: 0.75
    max_bubble: 1.0
    contraction_gain: 1.0
    repulsion_gain: 2.0
    damping_gain: 0.75
    cycle_count: 128
    desired_speed: 0.2
    advance_distance: 0.3

/kurome/pose_publisher:
  ros__parameters:
    odom_in: "odom"
    produce_map_frame: true
    odom_pose_out: "odom_pose"
    map_pose_out: "map_pose"

/kurome/local_mask:
  ros__parameters:
    scan_in: "/scan"
    scan_out: "scan"
    cull_dist: 0.75

